LandSandBoat/server
# [sql] change swaps Despot and Faust skill lists

**_I affirm:_**
- [x[ I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Essentially "swaps" Despot and Faust skills back to what they should be.

## Steps to test these changes

1. Go to Sky
2. SpawnMob on Despot
3. Fight until you see he uses Typhoon, frown
4. Go to Shrine
5. SpawnMob on Faust
6. Fight until you see he uses Panzerfaust, frown deepens.
